app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java

```diff
@@ -270,4 +276,23 @@ public static boolean isHardwareCameraInstalled(Context context) {
         }
         return false;
     }
+
+    public static OkHttpClient HttpClientBuilder() {
```

Comment:
I like the approach; but I'd like the TLS 1.2 and allowed CipherSuites to be only limited to API 24 (higher API levels might want to use TLS 1.3 for example)